docs/sphinx/_toc.yml.in

```diff
@@ -19,3 +19,13 @@ subtrees:
   - caption: About
     entries:
     - file: license
+  - caption: Samples
+    entries:
+    - file: samples/client_extop_softmax
+      title: client_extop_softmax.cpp
+    - file: samples/sample_hipblaslt_ext_op_layernorm
+      title: sample_hipblaslt_ext_op_layernorm.cpp
+    - file: samples/sample_hipblaslt_ext_op_amax
+      title: sample_hipblaslt_ext_op_amax.cpp
+    - file: samples/sample_hipblaslt_ext_op_amax_with_scale
+      title: sample_hipblaslt_ext_op_amax_with_scale.cpp
```

Comment:
Can we move these samples as a sub-tree of line #17 (a subtree of the actual ext-ops doc instead)?  


- file: ext-ops
   subtrees:
   - captions: Samples
   ...